pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/readers/PinotSegmentColumnReaderImpl.java

```diff
@@ -200,43 +201,43 @@ public byte[] nextBytes() {
   }
 
   @Override
-  public int[] nextIntMV() {
+  public MultiValueResult<int[]> nextIntMV() {
     if (!hasNext()) {
       throw new IllegalStateException("No more values available");
     }
     int[] value = _segmentColumnReader.getIntMV(_currentIndex);
     _currentIndex++;
-    return value;
+    return MultiValueResult.of(value, null);
```

Comment:
lets document that pinot doesn't support nulls in MV and thus we're passing null to validity vector